server2/models/user.js

```diff
@@ -12,7 +18,7 @@ module.exports = User = sequelize.define('User', {
     allowNull: false,
     primaryKey: true
```

Comment:
why name is the primary key of the user?

Reply:
emmmm id is set to pk?